extensions/vscode/webviews/homeView/src/components/tree/TreeItem.vue

```diff
@@ -35,7 +35,9 @@
         :class="codicon"
       />
       <div class="tree-item-label-container">
-        <span class="tree-item-title">{{ title }}</span>
+        <span class="tree-item-title" :data-automation="`req`">{{
```

Comment:
I'm not understanding why you are evaluating an expression which just returns the string of "req"? Or do I have this wrong? I think it should be able to be replaced with:
```
<span class="tree-item-title" data-automation="req">{{
```

Reply:
i think i had been using a merge field there but then realized i didn't need it but forgot to remove the "`".  Thanks, I'll fix that.